### DISCUSSION TITLE
How do I add an optional command line startup parameter?

### DISCUSSION THREAD

**Discussion by domenicdistefano** [Ch4r0ne/ARK-Ascended-Server-Manager#28] [Q&A]
Don't see anything in the GUI to add a command line / startup option. Any way to manually add it to the startup?

Thanks!

**Comment by kicker300zx** [comment]
did you ever figure this out or get an answer?

**Comment by Ch4r0ne** [comment]
@domenicdistefano you can add start paremther in the powershell skirpt at line 942, unfortunately at the moment I have not yet built a solution into the GUI. But if I find the time I will add it. 

$ServerArguments = [System.String]::Format('start {0}?listen?SessionName="{1}"?Port={2}?QueryPort={3}?ServerPassword="{4}"?RCONEnabled={7}?RCONPort={8}?ServerAdminPassword="{5}" -{9} -automanagedmods -mods={10}, -WinLiveMaxPlayers={6}, -{11}, -ServerPlatform={12}', $ServerMAP, $ServerName, $Port, $QueryPort, $Password, $AdminPassword, $MaxPlayers, $RCONEnabled, $RCONPort, $BattleEye, $Mods, $ForceRespawnDinosValue, $ServerPlatform)
